core/new-gui/src/app/dashboard/component/top-bar/user-icon/user-icon.component.ts

```diff
@@ -23,7 +23,6 @@ export class UserIconComponent {
     private modalService: NgbModal,
     private userService: UserService
   ) {
-
```

Comment:
please remove this line change from the PR

Reply:
done